src/components/OptionsList.js

```diff
@@ -196,7 +200,7 @@ class OptionsList extends Component {
 
     render() {
         return (
-            <View style={[styles.flexGrow0]}>
+            <View style={[...(this.props.listContainerStyles ? this.props.listContainerStyles : [styles.flex1])]}>
```

Comment:
I think this can be rewritten as 
```suggestion
            <View style={[...(this.props.listContainerStyles || [styles.flex1])]}>
```

Reply:
Though I'm a little unsure about the use of spread syntax here. Wouldn't `[...(x)]` be equal to just `x`?

I think this can be further simplified to:
 
```suggestion
            <View style={this.props.listContainerStyles || [styles.flex1]}>
```

Is there a specific reason you'd want to have the syntax in comment right above isntead?

Reply:
yeah if we want to make this _super_ clean we can just do this

```js
const defaultProps = {
    listContainerStyles: [styles.flex1],
};

...
<View style={this.props.listContainerStyles}
```

Reply:
@marcaaron I love this clean solution! Will implement it and push it. Thank you!